recipes/beyla-golden-signals/beyla-daemonset.yaml

```diff
@@ -33,6 +33,29 @@ spec:
     spec:
       serviceAccountName: beyla
       hostPID: true
+      initContainers:
+        - name: mount-bpf-fs
+          image: grafana/beyla:1.5.0
+          args:
+          # Create the directory using the Pod UID, and mount the BPF filesystem.
+          - 'mkdir -p /sys/fs/bpf/$BEYLA_BPF_FS_PATH && mount -t bpf bpf /sys/fs/bpf/$BEYLA_BPF_FS_PATH'
```

Comment:
Again not a ton of context, but the docs for `Bidirectional` have this scary warning

> [... In addition, any volume mounts created by containers in pods must be destroyed (unmounted) by the containers on termination.](https://kubernetes.io/docs/concepts/storage/volumes/#volume-types:~:text=In%20addition%2C%20any%20volume%20mounts%20created%20by%20containers%20in%20pods%20must%20be%20destroyed%20(unmounted)%20by%20the%20containers%20on%20termination.)

Do we need to clean up this mount?

Reply:
I think it is OK not to clean up as long as we re-use the mount when it is redeployed.  Leaking a single mount shouldn't be a huge deal.

Reply:
Ya that makes sense, I see you updated to use `KUBE_NAMESPACE` instead of pod UID so it should be persistent